### PR TITLE
feat(dli): add new datasource to get list of sql jobs

### PR DESCRIPTION
**Pull request by wangyuancheng01**
**What this PR does / why we need it**:
   add new datasource to get list of sql jobs.

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST="./huaweicloud/services/acceptance/dli" TESTARGS="-run TestAccDataSourceDliSqlJobs_basic"
...
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dli -v -run TestAccDataSourceDliSqlJobs_basic -timeout 360m -parallel 4
=== RUN   TestAccDataSourceDliSqlJobs_basic
=== PAUSE TestAccDataSourceDliSqlJobs_basic
=== CONT  TestAccDataSourceDliSqlJobs_basic
--- PASS: TestAccDataSourceDliSqlJobs_basic (430.01s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dli       430.101s
```

### CODE REVIEW

File: docs/data-sources/dli_sql_jobs.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+subcategory: "Data Lake Insight (DLI)"
+layout: "huaweicloud"
+page_title: "HuaweiCloud: huaweicloud_dli_sql_jobs"
+description: |-
+  Use this data source to get the list of the DLI sql jobs.
+---
+
+# huaweicloud_dli_sql_jobs
+
+Use this data source to get the list of the DLI sql jobs.
```
Comment (Lance52259) [inline docs/data-sources/dli_sql_jobs.md]:
```suggestion
Use this data source to get the list of the DLI SQL jobs.
```

File: docs/data-sources/dli_sql_jobs.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+* `sql` - The SQL statement is executed by the job.
+
+* `start_time` - The time when a job is start.
+
+* `end_time` - The time when a job is end.
+
+* `duration` - The job running duration (unit: millisecond).
+
+* `tags` - The label of the job.
```
Comment (Lance52259) [inline docs/data-sources/dli_sql_jobs.md]:
```suggestion
* `tags` - The key/value pairs to associate with the job.
```

File: docs/data-sources/dli_sql_jobs.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+* `owner` - The user who submits the job.
+
+* `database_name` - The database name where the table that records its operations is located.
+
+* `sql` - The SQL statement is executed by the job.
+
+* `start_time` - The time when a job is start.
+
+* `end_time` - The time when a job is end.
```
Comment (Lance52259) [inline docs/data-sources/dli_sql_jobs.md]:
```suggestion
* `start_time` - The time when a job is start, in RFC 3339 format.

* `end_time` - The time when a job is end, in RFC 3339 format.
```

File: docs/data-sources/dli_sql_jobs.md (hunk 1 above)
Comment (wangyuancheng01) [reply docs/data-sources/dli_sql_jobs.md]: done

File: docs/data-sources/dli_sql_jobs.md (hunk 2 above)
Comment (wangyuancheng01) [reply docs/data-sources/dli_sql_jobs.md]: done

File: docs/data-sources/dli_sql_jobs.md (hunk 3 above)
Comment (wangyuancheng01) [reply docs/data-sources/dli_sql_jobs.md]: done

File: docs/data-sources/dli_sql_jobs.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) Specifies the region in which to query the resource.
+  If omitted, the provider-level region will be used.
+
+* `job_id` - (Optional, String) Specifies the ID of the job to be queried.
+
+* `type` - (Optional, String) Specifies the type of the job to be queried.
```
Comment (Lance52259) [inline docs/data-sources/dli_sql_jobs.md]:
```suggestion
* `type` - (Optional, String) Specifies the type of the jobs to be queried.
```

File: docs/data-sources/dli_sql_jobs.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+* `status` - (Optional, String) Specifies the status of the job to be queried.
+  The valid values are **FINISHED**, **FAILED** and **CANCELED**.
+
+* `queue_name` - (Optional, String) Specifies the queue name which this job to be submitted belongs.
```
Comment (Lance52259) [inline docs/data-sources/dli_sql_jobs.md]:
```suggestion
* `queue_name` - (Optional, String) Specifies the queue name which the jobs to be submitted belong.
```

Comment (wangyuancheng01) [reply docs/data-sources/dli_sql_jobs.md]: done

File: docs/data-sources/dli_sql_jobs.md (hunk 4 above)
Comment (wangyuancheng01) [reply docs/data-sources/dli_sql_jobs.md]: done

**Pull request merged**